### ISSUE TITLE
Cann't use on mac os system

### ISSUE THREAD

**Issue by brightgong** [michalrus/git-hooks-code-autoformat#6]
I use this hook on my mac and it didn't work when i done everything written in README step by step.<br/>
The error is: <strong>find: -executable: unknown primary or operator </strong>

did i make mistake on somewhere?

**Comment by rvvincelli** [comment]
In the `ejf` binary `sed -re` is invoked, but the BSD-style should be `sed -Ee`. Anyway I get a runtime exception in the main when running the formatter directly from the command line.

**Comment by alxpgr** [comment]
The problem is that -executable is GNU specific. This [thread on StackExchange](http://apple.stackexchange.com/questions/116367/find-all-executable-files-within-a-folder-in-terminal) gives the alternative that should work in both systems. I tried this change and the change for sed as indicated by @rvvincelli and it works on my mac. I haven't been able to test it on a Linux machines though.

**Comment by michalrus** [comment]
Would any of you care to submit a PR, that works on both systems, GNU, and *the other one*? :grin:

**Comment by tianheng** [comment]
a quick fix:
1. `vim git-hooks/tools/eclipse-java-formatter/ejf`
2. modify `sed -re` to `sed -Ee`

**Comment by michalrus** [comment]
I see in the GNU sed’s manual:
```
     -E, -r, --regexp-extended

          use  extended regular  expressions in  the script  (for
          portability use POSIX -E).
```
… so we can just switch to `-Ee` everywhere.

**Comment by michalrus** [comment]
Since I don’t have a non-GNU system to test, it would be optimal if someone else did a PR, but here it goes:

**Comment by michalrus** [comment]
(And that `-executable` bit was fixed in #10.)
